core/trino-main/src/main/java/io/trino/sql/planner/planprinter/JsonRenderer.java

```diff
@@ -127,30 +124,5 @@ public List<JsonRenderedNode> getChildren()
         {
             return children;
         }
-
-        @Override
```

Comment:
Removal of these methods looks unrelated to this commit, so let's do it in a prep commit